### PR TITLE
Feat aya slider

### PR DESCRIPTION
**Pull request by tutanck**
This PR contains the changes related to the slider's style (navigation arrows)

### CODE REVIEW

File: components/slides/Slider.vue [hunk 1]
```diff
@@ -1,59 +1,70 @@
...
 
-.left:hover .triangle {
-	border-right: 5vw solid #EBB815;
-}
-
-.right:hover .triangle {
-	border-left: 5vw solid #EBB815;
+.reversed-arrow {
+	/* TODO: Check Prefixer */
+	/* Rotate arrow */
```
Comment (nbrdx) [inline components/slides/Slider.vue]: remove comment

File: components/slides/-Slider.spec.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+			mocks: {
+				$route: {
+					fullPath: '/vision/intro'
+				},
+				$router: { push: jest.fn() }
+			}
+		});
+	});
+
+	describe('Properties validation rules', () => {
```
Comment (FBerthelot) [inline components/slides/-Slider.spec.js]: Useless test, non ?

File: components/slides/-Slider.spec.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+		});
+
+		it('has a path props of type String', () => {
+			const path = cmp.vm.$options.props.path;
+			expect(path.type).toBe(String);
+		});
+	});
+
+	describe('HTML content', () => {
+		it('has one div with class content', () => {
```
Comment (FBerthelot) [inline components/slides/-Slider.spec.js]: idem

File: components/slides/-Slider.spec.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+			expect(cmp.find('div').classes()).toContain('content');
+		});
+
+		it('has one svg and one link snippet', () => {
```
Comment (FBerthelot) [inline components/slides/-Slider.spec.js]: idem

**Pull request merged**